packages/camera/package.js

```diff
@@ -0,0 +1,22 @@
+Package.describe({
+  name: "mikkelking:camera",
```

Comment:
I think there is a bit of a confusion on getting the package location in the repo.

Reply:
Yes, both the package name and package should be `mdg:camera`.